assets/js/script.js

```diff
@@ -10,39 +16,50 @@ const currentPos = {
 
 // ** For resizing the canvas depending on the window size.**
 const setDisplay=()=>{
-	const headerHeight = document.querySelector('header').offsetHeight;
-	const footerHeight = document.querySelector('header').offsetHeight;
-	canvas.height = window.innerHeight -headerHeight -footerHeight;
+	canvas.height = window.innerHeight -headerHeight -footerHeight - 9;
 	canvas.width = window.innerWidth;
 }
-window.addEventListener('load', setDisplay);
-window.addEventListener('resize', setDisplay);
+
+//Dividing the canvas into pseudo grids.
+// Testing incrementing x and y pos of apple in increments of 30;
+
+// Number of X positions possible at intervals of 30: (0,30,60,90,.....)
+const xGridPositions = () => {
+  if (canvas.width / canvasElementsDim !== Math.floor(canvas.width / canvasElementsDim)) {    // Checking if the canvas width is exactly divisible by 30
+    xPositions = Math.floor(canvas.width / canvasElementsDim) - 1;             // If not then 30px box can't fit without overflow, we remove 1 position to make sure the box fits even if there is extra space at the right and bottom sides.
+                                                                // Ex: canvas width = 1545 then if condition becomes true since a box cannot fit in 15 px we have xGridPostions as 50 - 1 to accomodate the last box. 
+                                                                // The total usable canvas width now becomes 1530px;
```

Comment:
Need to align game area in center for better experience

Reply:
That will be the next step... I pushed this intermediate stage so that we can know what we will be dealing with and what my though process is and that we have an agreement in this  direction.